samples/nrf9160/nrf_cloud_rest_device_message_tol2/src/main.c

```diff
@@ -0,0 +1,153 @@
+/*
+ * Author: tol2@nordicsemi.no
+ * Date: 07/28/2022
+ * NRFCloud rest message sample app replicate
+ */
+
+#include <zephyr/kernel.h>
+#include <modem/nrf_modem_lib.h>
+#include <nrf_modem_at.h>
+#include <modem/modem_info.h>
+#include <zephyr/settings/settings.h>
+#include <net/nrf_cloud.h>
+#include <net/nrf_cloud_rest.h>
+#include <zephyr/sys/reboot.h>
```

Comment:
Is anything in <zephyr/sys/reboot.h> used?  Maybe you can remove it.

Reply:
I will remove that, too.

Reply:
Actually, there are some more unnecessary libraries, I'll remove them as well.